docs/openapi.yml

```diff
@@ -1899,6 +1899,11 @@ components:
           description: regex of paths to be ignored for changes
           type: array
           example: [ "/target", "/.m2", "*/.git/*" ]
+        refPaths:
+          description: additional referenced paths to monitor
```

Comment:
Should we update the description to mention that it is used for including files that are not within the project folder?

Reply:
Updated in https://github.com/eclipse/codewind/pull/1535/commits/63672f46ef89d3518536258b818436f19bb52927